### PR TITLE
Update for Peer Rationale Reflection Tool

### PR DESCRIPTION
**Pull request by jleong-openedx**
## [DOC-3502](https://openedx.atlassian.net/browse/DOC-3502)

Update documentation for the 0.6 release of the Peer Instruction Tool, now known as the Peer Rationale Reflection Tool. Pull request for code changes [here](https://github.com/edx/edx-platform/pull/14041).

### Date Needed

Hopefully the documentation changes can be accepted along with the code changes by January 2017 in time for the start of term.

### Reviewers

- [x] Doc team review (copy edit): @shaunagm, @catong 

FYI: @sstack22 

### Testing

- [x] Executed ./run_tests.sh without any errors

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [x] If multiple commits, squash before merge.

Change name from Peer Instruction to Peer Rationale Reflection. Also
update screenshots and describe scenario with no correct answer.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @jleong-openedx! I've created [OSPR-1598](https://openedx.atlassian.net/browse/OSPR-1598) to keep track of it in JIRA, where we prioritize reviews. Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: en_us/shared/exercises_tools/peer_rationale_reflection_tool.rst [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+###################################
+Peer Rationale Reflection Tool
+###################################
 
 .. note:: EdX offers full support for this tool.
 
 The Peer Instruction learning system provides students with in class
 opportunities to discuss questions and arrive at a deeper understanding of
-concepts. The peer instruction tool emulates this classroom experience for the
+concepts. The Peer Rationale Reflection Tool emulates this classroom experience for the
```
Comment (catong) [inline en_us/shared/exercises_tools/peer_rationale_reflection_tool.rst]: @jleong-openedx Lowercase for the name of tools: "The peer rationale reflection tool..."

File: en_us/shared/exercises_tools/peer_rationale_reflection_tool.rst [hunk 2]
```diff
@@ -24,7 +24,7 @@ resources of the `Mazur Group`_ or the `Turn to Your Neighbor`_ blog.
 Assignment Overview
 *********************
 
-Assignments created with the peer instruction tool present learners with a
+Assignments created with the Peer Rationale Reflection Tool present learners with a
```
Comment (catong) [inline en_us/shared/exercises_tools/peer_rationale_reflection_tool.rst]: Lowercase tool name throughout, please

File: en_us/shared/exercises_tools/peer_rationale_reflection_tool.rst [hunk 3]
```diff
@@ -83,43 +83,44 @@ for the exercise.
...
- discussion component after the peer instruction component to give learners an
- opportunity to continue the conversation.
+.. note:: You might consider including an HTML component before the Peer
+ Rationale Reflection component to describe the workflow that learners will
+ experience in this assessment type. You might also consider including a
+ content-specific discussion component after the Peer Rationale Reflection
+ component to give learners an opportunity to continue the conversation.
 
- For more information about how learners experience peer instruction
+ For more information about how learners experience Peer Rationale Reflection
```
Comment (catong) [inline en_us/shared/exercises_tools/peer_rationale_reflection_tool.rst]: No initial caps for "Peer Rationale Reflection"

File: en_us/shared/exercises_tools/peer_rationale_reflection_tool.rst [hunk 4]
```diff
@@ -83,43 +83,44 @@ for the exercise.
...
+Reflection assignment, you must enable the Peer Rationale Reflection Tool for
+your course.
 
-To enable the peer instruction tool in Studio, you add the ``"ubcpi"`` key to
-the **Advanced Module List** on the **Advanced Settings** page. (Be sure to
-include the quotation marks around the key value.) For more information, see
-:ref:`Enable Additional Exercises and Tools`.
+To enable the Peer Rationale Reflection Tool in Studio, you add the
+``"ubcpi"`` key to the **Advanced Module List** on the **Advanced Settings**
+page. (Be sure to include the quotation marks around the key value.) For more
```
Comment (catong) [inline en_us/shared/exercises_tools/peer_rationale_reflection_tool.rst]: This was in the existing doc, but no need for parentheses and strike "the" before "quotation marks": "...on the **Advanced Settings** page. Be sure to include quotation marks around the key value."

File: en_us/shared/exercises_tools/peer_rationale_reflection_tool.rst [hunk 5]
```diff
@@ -1,16 +1,16 @@
...
-##########################
-Peer Instruction Tool
-##########################
+###################################
+Peer Rationale Reflection Tool
+###################################
 
 .. note:: EdX offers full support for this tool.
 
 The Peer Instruction learning system provides students with in class
```
Comment (catong) [inline en_us/shared/exercises_tools/peer_rationale_reflection_tool.rst]: @jleong-openedx Should this intro still say "Peer Instruction learning system"?

File: en_us/shared/exercises_tools/peer_rationale_reflection_tool.rst [hunk 6]
```diff
@@ -128,25 +129,25 @@ Upload Images (Optional)
...
 
 #. Under **Add New Component**, select **Advanced**, and then select **Peer
-   Instruction**. Studio adds the new component to the unit and uses a template
-   to supply a sample question, answers, and so on.
+   Rationale Reflection**. Studio adds the new component to the unit and uses
+   a template to supply a sample question, answers, and so on.
 
-   .. image:: ../../../shared/images/peer_instruction_studio.png
-     :alt: A peer instruction assignment in Studio that uses the template.
+   .. image:: ../../../shared/images/peer_rationale_reflection_studio.png
```
Comment (catong) [inline en_us/shared/exercises_tools/peer_rationale_reflection_tool.rst]: @jleong-openedx This image seems very faint/greyed out. If you have a better image, please substitute it.

File: en_us/shared/exercises_tools/peer_rationale_reflection_tool.rst [hunk 7]
```diff
@@ -165,11 +166,11 @@ assignment.
 Add the Assignment Details
 ============================================
 
-Before you add the details of the assignment to a peer instruction component,
-you :ref:`design the assignment<Designing a Peer Instruction Assignment>` by
-preparing a question, answer choices, and other required elements. If the
+Before you add the details of the assignment to a Peer Ratioanle Reflection
```
Comment (catong) [inline en_us/shared/exercises_tools/peer_rationale_reflection_tool.rst]: typo: "Rationale". (Also remove initial caps)

**Review by catong — CHANGES_REQUESTED** [review]
@jleong-openedx Just minor edits. Thanks!

File: en_us/shared/exercises_tools/peer_rationale_reflection_tool.rst (hunk 5 above)
Comment (jleong-openedx) [reply en_us/shared/exercises_tools/peer_rationale_reflection_tool.rst]: Yes, as that is the method that the Peer Rationale Reflection Tool (PRRT) is based upon, not our tool itself.

File: en_us/shared/exercises_tools/peer_rationale_reflection_tool.rst (hunk 1 above)
Comment (jleong-openedx) [reply en_us/shared/exercises_tools/peer_rationale_reflection_tool.rst]: This is now corrected.

File: en_us/shared/exercises_tools/peer_rationale_reflection_tool.rst (hunk 2 above)
Comment (jleong-openedx) [reply en_us/shared/exercises_tools/peer_rationale_reflection_tool.rst]: This is now corrected.

File: en_us/shared/exercises_tools/peer_rationale_reflection_tool.rst (hunk 3 above)
Comment (jleong-openedx) [reply en_us/shared/exercises_tools/peer_rationale_reflection_tool.rst]: This is now corrected.

File: en_us/shared/exercises_tools/peer_rationale_reflection_tool.rst (hunk 4 above)
Comment (jleong-openedx) [reply en_us/shared/exercises_tools/peer_rationale_reflection_tool.rst]: This is now corrected.

File: en_us/shared/exercises_tools/peer_rationale_reflection_tool.rst (hunk 6 above)
Comment (jleong-openedx) [reply en_us/shared/exercises_tools/peer_rationale_reflection_tool.rst]: I don't believe we can generate a better image, but we will keep this in mind and replace it if we can.

File: en_us/shared/exercises_tools/peer_rationale_reflection_tool.rst (hunk 7 above)
Comment (jleong-openedx) [reply en_us/shared/exercises_tools/peer_rationale_reflection_tool.rst]: This is now corrected.

**Comment by jleong-openedx** [comment]
Thank you @catong for the feedback and corrections; please see the latest commit.

**Review by catong — APPROVED** [review]

**Comment by catong** [comment]
@jleong-openedx Thank you! This doc PR is now ready to be merged when the corresponding code PR merges.

**Comment by catong** [comment]
Oh, please squash commits.

**Comment by jleong-openedx** [comment]
Okay, thank you @catong; should I keep the update to AUTHORS as a separate commit (as stated [here](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst))?

**Comment by catong** [comment]
@jleong-openedx Sure, keep the AUTHORS update as a separate commit. Thanks!

**Comment by jleong-openedx** [comment]
I have just squashed the commits, and kept the AUTHORS update as a separate commit. Thank you @catong!

**Comment by jleong-openedx** [comment]
Hi @catong, I have just updated this to revert Peer Rationale Reflection to Peer Instruction, as a decision has been made to stay with the old name for now.

**Comment by catong** [comment]
Thank you for the update, @jleong-openedx.

**Comment by gsong** [comment]
Waiting for associated PRs to complete.

**Comment by gsong** [comment]
@catong FYI, the corresponding PR edx/edx-platform#14041 will be merged soon, so you'll want to keep an eye out for it.

**Comment by catong** [comment]
Thanks @gsong, will do.

**Comment by catong** [comment]
Merging https://github.com/edx/edx-documentation/pull/1354

**Comment by jleong-openedx** [comment]
Thank you @catong!

**Pull request merged**